extensions/cornerstone/src/services/ToolGroupService/ToolGroupService.ts

```diff
@@ -316,16 +288,17 @@ export default class ToolGroupService {
   }
 
   private _addTools(toolGroup, tools, configs) {
-    const toolNames = this._getToolNames(tools);
-    toolNames.forEach(toolName => {
-      // Initialize the toolConfig if no configuration is provided
-      const toolConfig = configs[toolName] ?? {};
-
-      // if (volumeUID) {
-      //   toolConfig.volumeUID = volumeUID;
-      // }
-
-      toolGroup.addTool(toolName, { ...toolConfig });
+    ['active', 'passive', 'enabled', 'disabled'].forEach(toolStateName => {
```

Comment:
can we use enums for these?

Reply:
Updated to use the Enum from `cornerstoneTools`